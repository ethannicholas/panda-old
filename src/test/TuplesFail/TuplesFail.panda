var x:(Int) := (5)