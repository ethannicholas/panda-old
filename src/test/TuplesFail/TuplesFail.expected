error: TuplesFail.panda:1:17: expected '(panda.core.Int32)', but found 'panda.core.Int8'
1 error
